### PR TITLE
Allow to unban address from API

### PR DESCRIPTION
**Pull request by mmadariaga**
Added [DELETE] /banned_addresses/antibruteforce/{id} brand API endpoint

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/admin/application/controllers/KlearCustomBruteForceUnbanController.php [hunk 1]
```diff
@@ -32,15 +31,19 @@ public function indexAction()
...
+                $dataGateway->remove(
+                    BannedAddress::class,
+                    [$pk]
+                );
 
-            $msg = $result ? "Address successfully unbanned" : "Problems unbanning address";
+                $msg = "Address successfully unbanned";
+            } catch (\Exception $e) {
+                $msg = "Problems unbanning address";
+            }
```
Comment (Kaian) [inline web/admin/application/controllers/KlearCustomBruteForceUnbanController.php:46]: what about translating these ones like title variable below 

```
 $msg = $this->_helper->translate(
    "Address successfully unbanned"
 );

Comment (mmadariaga) [reply web/admin/application/controllers/KlearCustomBruteForceUnbanController.php]: Done

**Comment by ironArt3mis** [comment]
Rebase completed.

**Review by cruzccl — APPROVED** [review]
Epic!

**Pull request merged**